src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
```

Comment:
According to Code Style Rules, you should avoid using fixed width. Try to make your block's width flexible.